modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -39,15 +39,15 @@ spec:
           pod: {}
         - name: bridge-net <1>
           multus:
-            networkName: <network-namespace/a-bridge-network> <2>
+            networkName: default/<a-bridge-network> <2>
```

Comment:
This example implies that the NAD must always exist in the default network, which is not true. The example should clarify the flexibility in the NAD location, which can either be "default" namespace or the same namespace as the VM.

Reply:
Does the note on line 50 clarify this flexibility?

Reply:
I'm afraid not. The way that "default" is written here makes the impression that it is a mandatory value.

Reply:
Following our discussion:
I would actually go with the original phrasing, almost as is:
`networkName: <network-namespace>/<a-bridge-network> <2>`
Note that both `network-namespace` and `a-bridge-network` are put in angle brackets.
And then, in callout <2>, I would say something like:
<network-namespace> is the namespace where the Network Attachment Definition exists; the only 2 possible options for this namespace is either `default` namespace or the same namespace where the VM is to be created.